website/content/gateway/api_reference/resources/account/account.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Account"
+date: 2022-04-13T12:37:22+02:00
+anchor: "account"
+weight: 185
+---
+#### Account
+The account resource holds basic merchant account information. Only `HTTP GET` is supported for this endpoint.
+```shell
+GET https://gateway.clearhaus.com/account
+```
+##### Response parameters
+
+{{% description_list %}}
+{{% description_term %}}merchant_id{{% regex %}}[0-9]{15}{{% /regex %}}{{% /description_term %}}
+{{% description_details %}}Used for 3-D Secure and also for reference when talking to our support staff. For 3-D Secure it is important to represent this number with leading zeros. 
+{{% /description_details %}}
+
+{{% description_term %}}descriptor{{% regex %}}[\x20-\x7E]{0,22} [ASCII printable characters](https://en.wikipedia.org/wiki/ASCII#ASCII_printable_characters) {{% /regex %}}{{% /description_term %}}
+{{% description_details %}}The default `text_on_statement`. 
```

Comment:
```suggestion
{{% description_details %}}The default `text_on_statement`.
```

Since you're using VSCode, I suggest setting `"files.trimTrailingWhitespace": true,` in `settings.json` as this issue appears several places.

Keep in mind that if you do use the above-mentioned setting, you can't use double-space at the end of the line to force a hard line break in Markdown. Luckily, this syntax hasn't been used anywhere in this project.